.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+  * [Open Source Contribution Policy](#open-source-contribution-policy)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Feature Requests](#feature-requests)
+  * [Bug Reports](#bug-reports)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+  * [Translations](#translations)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [C Styleguide](#c-styleguide)
+
+
+## What should I know before I get started?
+### Open Source Contribution Policy
+[Version 0.3, 2015–11–18](https://medium.com/@jmaynard/a-contribution-policy-for-open-source-that-works-bfc4600c9d83#.i9ntbhmad)
+
+#### Policy
+
+We will accept contributions of good code that we can use from anyone.
+
+#### What this means
+
+ - “We will accept”: This means that we will incorporate your contribution into the project’s codebase, adapt it as needed, and give you full credit for your work.
+ -  “contributions”: This means just about anything you wish to contribute to the project, as long as it is good code we can use. The easier you make it for us to accept your contribution, the happier we are, but if it’s good enough, we will do a reasonable amount of work to use it.
+ - “of good code”: This means that we will accept contributions that work well and efficiently, that fit in with the goals of the project, that match the project’s coding style, and that do not impose an undue maintenance workload on us going forward. This does not mean just program code, either, but documentation and artistic works as appropriate to the project.
+ - “that we can use”: This means that your contribution must be given freely and irrevocably, that you must have the right to contribute it for our unrestricted use, and that your contribution is made under a license that is compatible with the license the project has chosen and that permits us to include, distribute, and modify your work without restriction.
+ - “from anyone”: This means exactly that. We don’t care about anything but your code. We don’t care about your race, religion, national origin, biological gender, perceived gender, sexual orientation, lifestyle, political viewpoint, or anything extraneous like that. We will neither reject your contribution nor grant it preferential treatment on any basis except the code itself. We do, however, reserve the right to tell you to go away if you behave too obnoxiously toward us.
+
+#### If Your Contribution Is Rejected
+
+If we reject your contribution, it means only that we do not consider it suitable for our project in the form it was submitted. It is not personal. If you ask civilly, we will be happy to discuss it with you and help you understand why it was rejected, and if possible improve it so we can accept it. If we are not able to reach an agreement, then you are free to fork our project, as with any Open Source project, and add your contribution to your fork.
```

Comment:
Not sure if this doesn't sound too much like "feel free to fork us, more forks == better". Forking without the intention to just develop a patch and then definitely merge it back, should be the last resort and not common practice. We should probably stress that a little more.